airflow/ui/src/pages/Variables/Variables.tsx

```diff
@@ -58,7 +58,7 @@ const columns: Array<ColumnDef<VariableResponse>> = [
     cell: ({ row: { original } }) => (
       <Flex justifyContent="end">
         <EditVariableButton editKey={original.key} />
-        <DeleteVariableButton deleteKey={original.key} />
+        <DeleteVariableModal deleteKey={original.key} />
```

Comment:
Is it not a bit mis-leading that you now call it "Modal" whereas the modal just comes after the button? Should we not name it "Button" as before and the button brings a modal for confirmation with it?

Reply:
So meaning: Rename `DeleteVariableModal` to `DeleteVariableButton` and just have the modal in the button code.

Reply:
Actually yes, let me rename it back.